packages/frontend/src/ee/features/aiCopilot/components/ChatElements/ToolCalls/AiChartToolCalls.tsx

```diff
@@ -3,12 +3,8 @@ import {
     type ApiCompiledQueryResults,
     assertUnreachable,
     CsvFileVizConfigToolArgsSchemaTransformed,
+    findFieldsToolArgsSchemaTransformed,
     generateQueryFiltersToolArgsSchemaTransformed,
-    isCsvFileVizConfigToolArgs,
-    isFindFieldsToolArgs,
-    isGenerateQueryFiltersToolArgs,
-    isTimeSeriesMetricVizConfigToolArgs,
-    isVerticalBarMetricVizConfigToolArgs,
```

Comment:
I don't understand the intent for removing these 🤔

Reply:
I see, bc that way you get the success and parsed values in one? 

if so, then you might be able to remove these _type guards_ from schemas.ts

Reply:
that's right, interesting, commit hook didn't warn me they weren't used anywhere else

Reply:
> I see, bc that way you get the success and parsed values in one?

it's because we were first checking (which was parsing under the hood) and then parsing again with a transformed schema, just one parse directly with transformed schema has the same effect

Reply:
it's because we intentionally skip this check in `common` - easy to miss these!